ts/store/actions/wallet/payment.ts

```diff
@@ -281,6 +281,13 @@ export const runStartOrResumePaymentActivationSaga = createStandardAction(
   "PAYMENT_RUN_START_OR_RESUME_PAYMENT_ACTIVATION_SAGA"
 )<RunStartOrResumePaymentActivationSagaPayload>();
 
+/**
+ * the psp selected for the payment
+ */
+export const pspSelectedForPaymentV2 = createStandardAction(
+  "PAYMENT_PSP_V2_SELECTED"
```

Comment:
❓**question**: don't we need an action to _deselect_  a psp? I'm thinking about potentially stale data on the UI.

Reply:
I guess this could be a bug, thank you
With this commit 7ed0e06fb47cee4db1886970bd9739a2bd5abd75 now the relative store value is reset when the psp list is requested (the step before the user picks a new one)